app/utils/scheduling.py

```
@@ -213,6 +219,17 @@ def update_camera(name, template, image_file=None):
                 os.rename(output_path, output_path.replace(".tmp.png", ".png"))
                 lsuc = True
 
+    url = template.get("url")
+
+    if lsuc is True:
+        update_last_screenshot_time(name)
+    else:
+        entry = throttle_cache.get(url)
```

Ruff (0.11.9)

Undefined name `throttle_cache`

(F821)